src/autopr/options.d

```diff
@@ -57,9 +60,13 @@ Options parseOpts ( string[] opts )
         // -------------
         "num-entries", "Amount of entries to fetch per query (max & default: 100)",
         &options.num_entries,
+        // -------------
+        "delay", "How many seconds to wait between each PR",
+        &options.delay_seconds
     );
 
     options.num_entries = min(100, options.num_entries);
+    options.delay_seconds = max(0, min(3600, options.delay_seconds));
```

Comment:
well, negative wait times don't make sense so `max()` normalizes those to 0.
And I figured more than an 1h must be a mistake so I capped it.

I suppose I could use `enforce()` for the min value instead...

Reply:
Or use an `unsigned` type? Not sure how the options handler will behave then. In any case the restrictions seem reasonable. Maybe change the `max()` with a warning? Since it seems a bit unnecessarily restrictive, it is just there to avoid mistakes.

Reply:
In any case I think an error should be raised to the user if a provided value is incorrect and a warning (at most) if it is suspicious.